Updates on Annex DPP (#695)
admin-shell-io/aas-specs: documentation/IDTA-01001/modules/ROOT/pages/annex/dpp.adoc

  "granularity": "Item",
  "dppSchemaVersion": "ENXXX:v1.0",
  "dppStatus": "Active",
  "lastUpdate": "2025-08-22T03:12:00Z",
  "economicOperatorId": "gxx:ppp456789",
  "facilityId": "gxx:xxx987654",
  "contentSpecificationIds": [
	  "https://admin-shell-io/idta/digitalpassport/Nameplate/1",
	  "https://admin-shell-io/idta/digitalpassport/TechnicalData/1"},
	  "https://admin-shell-io/idta/digitalpassport/HandoverDocumentation/2",    
	  "https://admin-shell-io/idta/digitalpassport/ProductCondition/1",
	  "https://admin-shell-io/idta/digitalpassport/MaterialComposition/1", 
	  "https://admin-shell-io/idta/digitalproductpassport/Circularity/1"
	] 
}
----

Although some attributes (see <<code-dpp-json>>) of this Submodel specification can be mapped to attributes in the Asset Administration Shell (AAS) metamodel, 
the formal recommendation is to instead use this predefined Submodel Template IDTA-02099 for this metadata. 
This template is derived from the SMT IDTA-02006-4-0 Nameplate. 
For the corresponding semantic identifiers, please refer to their respective specifications.

The payload required by EN 18223 for the DPP metadata maps to the Value-Only format of this Submodel. 

> Note: for the other Submodels used to build a DPP EN 18223 allows two different formats "compressed" and "explanded" with compressed being mandatory to be supported. 

Author: ```suggestion
Note: for the other Submodels used to build a DPP EN 18223 allows two different formats "compressed" and "expanded" with compressed being mandatory to be supported. 
```

== Metamodel Mapping

> In <<image-dpp-data>> an overview of the metamodel as defined in EN 18223 is shownl

Author: ```suggestion
In <<image-dpp-data>> an overview of the metamodel as defined in EN 18223 is shown.
```

}
----

So in combination it would be the "semanticId" of SMT IDTA-02035-3 that is "https://admin-shell.io/idta/CarbonFootprint/CarbonFootprint/1/0", 
(see <<code-batterypass-pcf-valueonly-json>>).
The second value to be standardized is the one for the contentSpecificationIds. 

> For details on "contentSpecificationIds" see <<UML_for_SML_contentSpecificationIds>>.

Author: ```suggestion
For details on "contentSpecificationIds" see Submodel Template Specification IDTA-02099.
```

In <<code-multi-language-property-value-only>> it is shown how a MultiLanguageProperty with idShort "title" would be serialized in AAS Value-Only. 
In <<code-multi-language-property-jtc24>> it is shown how the same example is serialized in EN 18223.

> I.e. in AAS it serialized as described in the following extract:

Author: ```suggestion
I.e. in AAS it  is serialized as described in the following extract:
```
